filebeat/module/system/_meta/docs.asciidoc

```diff
@@ -1,20 +1,28 @@
 == System module
 
-This module collects and parses logs created by system logging server of common Unix/Linux based
-distributions.
+This module collects and parses logs created by system logging server of common
+Unix/Linux based distributions.
 
+[float]
 === Compatibility
 
-This module was tested with logs from OSes like Ubuntu 12.04, Centos 7, macOS Sierra, and others.
+This module was tested with logs from OSes like Ubuntu 12.04, Centos 7, macOS
+Sierra, and others.
```

Comment:
The "and others" might come back to us as an issue in support, maybe it's better to leave it out.